README.md

```diff
@@ -796,6 +796,75 @@ fs.createReadStream('./audio.raw')
 ```
 
 
+## Google Cloud Translation API (Alpha)
+
+- [API Documentation][gcloud-translate-docs]
+- [Official Documentation][cloud-translate-docs]
+
+#### Using the all-in-one module
+
+```
+$ npm install --save google-cloud
+```
+
+```js
+var gcloud = require('google-cloud');
+var translate = gcloud.translate;
+```
+
+#### Using the Google Translate API module
```

Comment:
Should this be `Translation`? And should we be publishing a new packages called `@google-cloud/translation`?